UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -568,11 +568,12 @@ <h3>Pre-recorded audio</h3>
 					audiobook), accompanies text (embedded audio and video clips), or represents an alternative to the
 					text (synchronized text-audio playback).</p>
 
-				<p>Audiobooks created for mainstream use provide important access for many users with disabilities even
+				<p>Audiobo
+oks created for mainstream use provide important access for many users with disabilities even
```

Comment:
Audiobooks got split across lines here.

```suggestion
				<p>Audiobooks created for mainstream use provide important access for many users with disabilities even
```